### ISSUE TITLE
Parsing error with Latex

### ISSUE THREAD

**Issue by inflation** [mdx-js/eslint-mdx#454]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

"eslint-plugin-mdx": "2.0.5",

### Steps to reproduce

Any .mdx file with a latex section like:
```latex
$$
\begin{align*}
\mathcal{L} &= - \frac{1}{4} F_{\mu \nu} F^{\mu \nu} \\
    &\phantom{{}=}+ i \bar{\psi} \cancel{D} \psi + h.c. \\
    &\phantom{{}=}+ \bar{\psi}_i y_{ij} \psi_j \phi + h.c. \\
    &\phantom{{}=}+ |D_\mu \phi|^2 - V(\phi)
\end{align*}
$$
```

### Expected behavior

Parse normally

### Actual behavior

Parsing error: Could not parse expression with acorn: Unexpected token

The problem is at the `*` in `\begin{align*}`.

### Runtime

Node v18

### Package manager

pnpm

### OS

macOS

### Build and bundle tools

Next.js

**Comment by ChristianMurphy** [comment]
MDX does not support latex by default.
If you want to enable latex in math blocks, you'd need https://github.com/remarkjs/remark-math added as a plugin through a `.remarkrc` file  https://github.com/unifiedjs/unified-engine/blob/main/doc/configure.md

**Comment by inflation** [comment]
Good catch. Never noticed that, adding the plugin solved it.

**Comment by r4ai** [comment]
I added `remark-math` to the `plugins` array in `.remarkrc.js` as shown below, but the parse error still persists. Am I doing something wrong? I’d really appreciate it if you could let me know how to fix it.

`.remarkrc.js`:

```js
import remarkPresetLintConsistent from "remark-preset-lint-consistent";
import remarkPresetLintRecommended from "remark-preset-lint-recommended";
import remarkMath from "remark-math";
import remarkPresetLintMarkdownStyleGuide from "remark-preset-lint-markdown-style-guide";
import remarkPresetPrettier from "remark-preset-prettier";

export default {
  plugins: [
    remarkMath,
    remarkPresetLintConsistent,
    remarkPresetLintRecommended,
    remarkPresetLintMarkdownStyleGuide,
    remarkPresetPrettier,
  ],
};
```

`math.mdx`:

```sh
$$
\forall \epsilon > 0, \exists \delta > 0 \text{ s.t. } |x - a| < \delta \Rightarrow |f(x) - b| < \epsilon
$$
```

```sh
$ bun run eslint .

/home/r4ai/src/repos/github.com/r4ai/eslint-plugin-mdx-math/src/math.mdx
  0:54  error  Preprocessing error: Could not parse expression with acorn

✖ 1 problem (1 error, 0 warnings)
```

Minimal reproduction repository: [https://github.com/r4ai/eslint-plugin-mdx-math-issue](https://github.com/r4ai/eslint-plugin-mdx-math-issue)

**Comment by JounQin** [comment]
Please raise a new issue instead.

**Comment by r4ai** [comment]
> Please raise a new issue instead.

I’ve opened a new issue at [https://github.com/mdx-js/eslint-mdx/issues/576](https://github.com/mdx-js/eslint-mdx/issues/576).
